src/cargo/ops/cargo_new.rs

```diff
@@ -384,7 +384,7 @@ fn mk(config: &Config, opts: &MkOptions) -> CargoResult<()> {
     let path = opts.path;
     let name = opts.name;
     let cfg = global_config(config)?;
-    let ignore = ["target/\n", "**/*.rs.bk\n",
+    let ignore = ["target/\n", "**/*.rs.bk\n", "*.sw[op]\n",
```

Comment:
Can also `*-swp` files be added? that would look like `*[.-]sw[op]`. This would be useful for me, as Kate generates `.filename.kate-swp` files